user-guide/Reference/Technical_information/NatsCustodian.md

```diff
@@ -12,7 +12,9 @@ NatsCustodian acquires the IP addresses of DataMiner Agents in the DataMiner Sys
 
 The system incorporates several checks, including identification of new nodes, detection of deleted nodes, tracking of unreachable nodes, validation of configuration correctness, and ensuring the running status of the NATS process.
```

Comment:
everything looks fine but you forgot to include the "and ensuring the running status of the NATS process" part.

Reply:
Hi @mykebueno, thanks for the feedback. I am a bit confused where this should be included. Could you perhaps add a suggestion? Just click the "+" icon, select the "Add a suggestion" button, and make your suggested changes.
![image](https://github.com/SkylineCommunications/dataminer-docs/assets/113898085/90baa77d-f117-49aa-ba8a-6544c1b177c4)